module/zfs/spa_config.c

```diff
@@ -86,7 +86,7 @@ spa_config_load(void)
 	int err;
 
 #ifdef _KERNEL
-	if (zfs_autoimport_disable)
+	if (boot == B_FALSE && zfs_autoimport_disable)
```

Comment:
Let's make this simpler.  On Linux `spa_boot_init()` is dead code and `zfs_autoimport_disable` was deprecated in a prior release.  Let's go ahead and remove `zfs_autoimport_disable` and its associated documentation.

Reply:
@allanjude ^